R/input-date.R

```diff
@@ -153,6 +153,12 @@ datePickerDependency <- function(theme) {
   )
 }
 
+datePickerSassLayer <- function() {
+  sass::sass_file(
+    system_file(package = "shiny", "www/shared/datepicker/scss/build3.scss")
+  )
+}
```

Comment:
I think I'd prefer `Layer` to not be in the name since I'd expect it to return a `sass::sass_layer()`.

Reply:
Sure. I used that because 1. It's not clear to me what exactly a _sass layer is_ and 2. in bslib we already had `component_dependency_sass()` which compiles Sass but actually returns an HTMLDependency.